doc/contributor/quick-rails-testing.md

```diff
@@ -0,0 +1,183 @@
+# Quick rails testing
+
+This repository contains small Rails application for testing 
+in `test/truffle/ecosystem/rails-app` directory. 
+The test can be executed with `jt test ecosystem rails-app` in TruffleRuby home directory.
+When executed it prepares its environment, starts rails server, 
+runs few tests against JSON API, and shut downs the rails server.
+It requires Redis as a database to by running in default configuration on `localhost:6379`.
+
+The `jt test ecosystem rails-app` should be executed in truffleruby home, 
```

Comment:
"in truffleruby" -> "in TruffleRuby's" or "in the TruffleRuby" (match this to my previous comment on the phrasing).